src/UserSpaceInstrumentation/Trampoline.cpp

```diff
@@ -217,10 +217,10 @@ void AppendCallToEntryPayloadAndOverwriteReturnAddress(uint64_t entry_payload_fu
       .AppendBytes({0x48, 0xb8})
       .AppendImmediate64(entry_payload_function_address)
       .AppendBytes({0xff, 0xd0})
-      .AppendBytes({0x5f})
-      .AppendBytes({0x48, 0xb8})
+      .AppendBytes({0x58})
```

Comment:
FWIW: The old code machine code disassembles to
```
pop rdi
mov rax, addr
mov (rdi), rax
```
So it just swaps the roles of rax, rdi. Both are equivalent in this context. The PR is fine anyway - now the comment and the machine code are in sync.

Reply:
Makes sense, thanks for checking!